src/sentry/api/endpoints/internal_beacon.py

```diff
@@ -0,0 +1,59 @@
+import logging
+
+from django.conf import settings
+from rest_framework.response import Response
+
+from sentry import get_version, options
+from sentry.api.base import Endpoint
+from sentry.http import safe_urlopen
+from sentry.tasks.beacon import BEACON_URL
+
+logger = logging.getLogger("beacon")
+
+
+class InternalBeaconEndpoint(Endpoint):
```

Comment:
Do we automatically cover this for CORS and reject requests from unknown sources to avoid abuse and flooding?

Reply:
Yeah it behaves similar to our other endpoints and requires a csrf token